The configure script is correctly getting the CFLAGS needed to compile a plugin
for gstreamer and storing them in GST_BASE_CFLAGS but the Makefiles are never
making use of those.

We actually have to use AM_CPPFLAGS as AM_CFLAGS is used everywhere but on the
real compiling rule...

Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
---
 src/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Makefile.am b/src/Makefile.am
index 8cb51d1..6af5d91 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -5,6 +5,8 @@
 ##############################################################################
 plugin_LTLIBRARIES = libgstx170.la
 
+AM_CPPFLAGS = @GST_BASE_CFLAGS@
+
 ##############################################################################
 # for the next set of variables, rename the prefix if you renamed the .la,   #
 #  e.g. libgstplugin_la_SOURCES => libmysomething_la_SOURCES                 #
-- 
1.8.1.2

